Docs building still failing on gstreamer-gl-x11

```
Documenting gstreamer-gl-x11 v0.21.1
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/gstreamer-gl-x11-0.21.1/src/lib.rs:9:37
  |
9 | #![cfg_attr(all(not(doctest), doc), feature(doc_cfg))]
  |                                     ^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not document `gstreamer-gl-x11`
```

We can either disable documenting gstreamer-gl-x11, use nightly for documenting or RUSTC_BOOTSTRAP on gstreamer-gl-x11 for docs building.

--- a/python/servo/post_build_commands.py
+++ b/python/servo/post_build_commands.py
@@ -266,6 +266,11 @@ class PostBuildCommands(CommandBase):
                     else:
                         copy2(full_name, destination)
 
+        # Documentation build errors shouldn't cause the entire build to fail. This
+        # prevents issues with dependencies from breaking our documentation build,
+        # with the downside that it hides documentation issues.
+        params.insert(0, "--keep-going")
+
         env = self.build_env()
         returncode = self.run_cargo_build_like_command("doc", params, env=env, **kwargs)
         if returncode:

